### PR TITLE
WQP-1309

### PR DESCRIPTION
**Pull request by dsteinich**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+FROM usgswma/openjdk:debian-stretch-openjdk-11.0.2-ca27ea0b2bc949ed612aeb98a561de91a797b07f
+
+RUN apt-get update && \
+  apt-get install --no-install-recommends --no-upgrade curl netcat -y && \
+  rm -rf /var/lib/apt/lists/*
+
+COPY --chown=1000:1000 --from=build /build/target/wqp.jar app.jar
+
+USER $USER
```
Comment (ayan-usgs) [inline Dockerfile]: So, one thing that we're going to want to add to this Dockerfile is the script that an entrypoint script that allows the container to read in its environment variables when it comes up. An example of the new script can be found https://github.com/ayan-usgs/simple-hello-world/blob/test-app/docker-entrypoint.sh. I'd like to do more testing against it if there wasn't also a certain other problem with our builds running when they shouldn't, but I don't expect it to change very much at this point.

**Comment by ayan-usgs** [comment]
I'm not sure if you want to include any Jenkinsfiles for the build/deploy at this point. Don't have to though because this really about Dockerizing.

File: Dockerfile (hunk 1 above)
Comment (dsteinich) [reply Dockerfile]: Due to the churn, I intentionally left the build & deploy items out of this commit.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,9 @@
...
-See the wqpgateway.properties file for configurable properties.
-
-Security can be enabled by adding the following to the Tomcat's context.xml:
-
-```
-    <Parameter name="spring.profiles.active" value="default,swagger,internal" />
-    <Parameter name="oauthResourceKeyUri" value = "<<url for token_key>>"/>
-    <Parameter name="oauthResourceId" value="wqp"/>
-```
+docker network create --subnet=172.25.0.0/16 wqp
```
[the remark this answers is not in the document]
Comment (dsteinich) [reply README.md]: The network creation is one way to run locally in different containers without one large docker-compose.

**Pull request merged**